examples/sensors-viewer/src/assets/css/cast.css

```diff
@@ -0,0 +1,188 @@
+body {
+    margin: 0;
+    padding: 0;
+    width: 100%;
+    font-size: 14px;
+    font-weight: 500;
+    font-family: 'Montserrat', sans-serif !important;
+}
+button {
+    cursor: pointer;
+    transition: 0.3s ease all;
+    text-decoration: none !important;
+}
+.container-fluid.main-div-card {
+    padding: 30px 0;
+}
+.sensor-main-div {
+    text-align: center;
+    margin-bottom: 40px;
+    text-transform: uppercase;
+    font-weight: 600;
+    background: #39801d;
+    color: #ffffff;
```

Comment:
Hardcoding colors here may be hard to change later. You'll have to search all the instances of that color but you can't replace them all, because the same color could be used by both text and background.
You can use bootstrap color classes: https://getbootstrap.com/docs/4.4/utilities/colors/